<launch>
  <arg name="with_rviz" default="false"/>
  <arg name="device" default="/dev/ttyACM0"/>
  
  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find adi_driver)/urdf/adis16470_breakout.urdf'" />

  <node pkg="adi_driver2" name="convert_rpy_to_tf" type="convert_rpy_to_tf.py" output="screen" >
    <param name="device" value="$(arg device)"/>
    <param name="parent_frame" value="odom"/>
    <param name="child_frame" value="imu"/>
  </node>

  <node if="$(arg with_rviz)" pkg="rviz" type="rviz" name="rviz" args="-d $(find adi_driver2)/launch/imu.rviz" />
</launch>
